README.md

```diff
@@ -1 +1,100 @@
 # hybro
+
+ReactNative <-> Web invoke/events communication
+
+---
+
+## ReactNative API
+
+### Installation
+
+Install and link [react-native-webview](https://github.com/react-native-community/react-native-webview)
+
+```
+npm i react-native-webview
+react-native link react-native-webview
+```
+
+### Usage example
+
+```javascript
+import { HybroView, } from 'hybro/native';
+
+const packages = {
+    'react-native': require('react-native'),
+};
+
+const uri = __DEV__
+    ? 'http://localhost:8080/index.html'
+    : Platform.select({
+        android: 'file:///android_asset/www/index.html',
+        ios: './www/index.html',
+    });
+
+<HybroView
+    packages={packages}
+    source={{ uri }}
+    originWhitelist={['*']}
+    mixedContentMode={'always'}
+    allowUniversalAccessFromFileURLs
+/>
+```
+
+Assumes you build web app into `android/app/src/main/assets/www` and into `ios/www`.
+
+Add `ios/www` folder into Xcode project (also into `Copy Bundle Resources` Build Phase).
+
+For DEV don't forget to reverse 8080 port for Android emulator:
+
+`adb reverse tcp:8080 tcp:8080`
+
+
+### HybroView Props
+
+* packages - object of plugged packages with names to call from web side
+
+---
+
+## Web API
+
```

Comment:
Please add RN API description too

Reply:
`packages` - single property

Reply:
We should add example for `EventTarget' in RN part. it's not obvious at all